src/makeRobot.js

```diff
@@ -38,7 +38,59 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return 'name: '
+      + this.name + ', '
+      + 'chip version: '
+      + this.version + ', '
+      + 'wheels: ' + this.wheels;
```

Comment:
```suggestion
      return 'name: '
      + this.name + ', '
      + 'chip version: '
      + this.version + ', '
      + 'wheels: ' + this.wheels;
```
Use interpolation instead of concatenation here. It will be more short